### PR TITLE
Let Mods be lazy

### PR DESCRIPTION
**Pull request by mebrooks01**
Made it so if the kick reason is `username`, `pfp` or `status` it will autofill the rest so that it's easier to kick users for bad usernames and such
I'm sure there's a better way to makes this check but it works and it's not that bad (I think)

### CODE REVIEW

**Comment by mebrooks01** [comment]
forgot to lowercase oop
catte can add that when he murges though
right?

**Comment by mebrooks01** [comment]
also capitilized c in change oop

**Comment by cAttte** [comment]
yes

File: src/commands/kick.ts [hunk 1]
```diff
@@ -36,8 +36,11 @@ export default new Command({
             return message.channel.sendError("Rude! You can't kick other staff.")
 
         const image = args.consumeImage()
-        const reason = args.consumeRest()
+        let reason = args.consumeRest()
         if (!reason) return message.channel.sendError("You must provide a reason!")
+        if (reason === "username") reason = "Please Change your username and come back."
```
Comment (mebrooks01) [inline src/commands/kick.ts:41]: add .tolowercase()

File: src/commands/kick.ts [hunk 2]
```diff
@@ -36,8 +36,11 @@ export default new Command({
...
+        if (reason === "pfp") reason = "Please Change your profile photo and come back."
+        if (reason === "status") reason = "Please Change your status and come back."
```
Comment (mebrooks01) [inline src/commands/kick.ts:43]: add .tolowercase()
and make it "Please change your <> and come back"

**Comment by vaporrrr** [comment]
includes() not inclueds()

**Comment by mebrooks01** [comment]
ffs

**Comment by mebrooks01** [comment]
this code was supper shitty and there is no way to edit the commit on a pr

**Pull request closed without merging**